src/strings/zh-cn.json

```diff
@@ -1578,5 +1578,11 @@
     "ClearQueue": "清空队列",
     "StopPlayback": "停止播放",
     "Writers": "作者",
-    "ViewAlbumArtist": "查看专辑艺术家"
+    "ViewAlbumArtist": "查看专辑艺术家",
+    "HeaderSelectFallbackFontPath" : "选择备用字体路径",
+    "HeaderSelectFallbackFontPathHelp": "浏览或输入一个包含备用字体文件的路径用于渲染 ASS/SSA 字幕。",
+    "LabelFallbackFontPath": "备用字体文件路径：",
+    "LabelFallbackFontPathHelp": "指定一个包含备用字体文件的路径用于渲染 ASS/SSA 字幕。允许的最大字体总容量为 20 MB。推荐使用轻量且适合网络传输的 woff2 格式字体。除非字体显示不正确，否则将其留空。",
```

Comment:
Same here for the file size limit and suggestion to remove the guidance for leaving the field blank.

Also, is adding translations here okay for new source strings, or does Weblate not like that? I may be worrying too much here.

Reply:
> Also, is adding translations here okay for new source strings, or does Weblate not like that? I may be worrying too much here.

In addition to adding the basic English version when adding new strings, contributors can also add the language version they are familiar with. This is more direct and convenient than adding translation to Weblate after the PR is merged.